config.yaml.dist

```diff
@@ -69,7 +69,57 @@ features:
   # more info: https://github.com/kubernetes-incubator/metrics-server
   metrics_server:
     disable: false
-    
+
+  # Enable OpenID-Connect support in API server
+  # More info: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens
+  openid_connect:
+    enable: false
+
+    config:
+      # The URL of the OpenID issuer, only HTTPS scheme will be accepted. If
+      # set, it will be used to verify the OIDC JSON Web Token (JWT).
+      issuer_url: ""
+
+      # The client ID for the OpenID Connect client, must be set if
+      # oidc-issuer-url is set.
```

Comment:
Rename `oidc-issuer-url` to `issue_url`.

Reply:
✔️